net/znc/Makefile

```diff
@@ -290,6 +295,12 @@ CONFIGURE_ARGS += \
 	--enable-tdns \
 	--enable-zlib
 
+ifeq ($(CONFIG_ZNC_ICU),y)
+	CONFIGURE_ARGS += --enable-charset
+else
+	CONFIGURE_ARGS += --disable-charset
+endif
+
```

Comment:
Just replace the removed line with
`$(if $(CONFIG_ZNC_ICU), --enable-charset, --disable-charset))`